src/mongoose_cets_discovery.erl

```diff
@@ -7,7 +7,7 @@
 start_link(DiscoOpts) ->
     Res = cets_discovery:start_link(DiscoOpts),
     %% Ensure metrics are added after the disco start
-    mongoose_metrics_probe_cets:start(),
+    mongoose_instrument_probe_cets:start(),
```

Comment:
I don't see `stop` anywhere in the code, but it should be there.

Reply:
cets_discovery does not support restarting that process (or the list of tables would get lost). Or stopping.
mongoose_internal_databases only has init function, and does not support stopping (because modules depend on internal databases).
I think there is stop function, when stopping the whole MongooseIM application.

I could put it into terminate function of cets_discovery, but probably a wrong place to do it.
Or add stop for mongoose_internal_databases, but it would still be called manually.

There is `mongoose_instrument_probe_cets:stop/0`, but it is called from tests only.

Reply:
Why is `terminate` the wrong place? I did that in a few places. Even if it works only in tests, it saves us from accidentally forgetting to stop it in tests. I would really like to have a `stop` for every `start`. So that if I start `cets_discovery` and stop it (even manually e.g. in small tests), I can start it again without an error.

Reply:
My current recommendation would be to put `start` in `mongoose_internal_databases:init/0`, which could be renamed `start/0`, and then we could call `stop` in a new function called `mongoose_internal_databases:stop/0`. But it's not mandatory to do it in this PR.

Reply:
that is tricky, because instrumentation starts in the ejabberd_sup.
While internal_databases could be started before or after.
But if we put it after sup, ejabberd_router (started from ejabberd_sup) will complain, mnesia is not initialized.
Or we could make intenal_databases as sup worker and find a good place in the sup to start it, but sounds a bit like out of scope and could be a separate issue.